src/kaleidoscope/driver/led/Base.h

```diff
@@ -43,8 +49,25 @@ class Base {
     };
     return c;
   }
-  uint8_t getLedIndex(uint8_t key_offset) {
-    return 0;
+  static uint8_t getLedIndex(uint8_t key_offset) {
+
+    static_assert(sizeof(_LEDDriverProps::key_led_map) >=
+                  _LEDDriverProps::led_count,
+                  "The number of LEDs in the LED map must be greater or "
+                  "equal to the number of LEDs");
```

Comment:
I do not think this assertion is valid. For example, the Dygma Raise has _less_ in the map than the total number of LEDs, because it uses some of the LEDs for underglow.

Reply:
Good point. Will fix that.

Reply:
Fixed and pushed an update.